app.py

```diff
@@ -70,6 +71,29 @@ def viewPatientDetails():
 	return render_template("patients.html",patient_details=patient_details,pageTitle="patients details")
 
 
+## searching for patient deatils using patiendID
+@app.route("/searchPatient")
+def searchPatient(patientId): # takes only one input parameter i.e patientId
+
+        #create dataframe
+        conn_search = sqlite3.connect("hospital.db")
+        df=pd.read_sql_query("SELECT * FROM patients",conn_search)
+        final_df=df[df["ws_pat_id"]==patientId]
+        
+        #render dataframe as html
+        html = final_df.to_html()
```

Comment:
I've never combined pandas and SQL before. Are you sure this will work? Also, can we add styling (CSS) to the HTML file created by panda?